ramalama/console.py

```diff
@@ -0,0 +1,53 @@
+"""
+MIT License
+
+(C) 2024-2025 ramalama developers
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+"""
+import os
+import sys
+import locale
+import logging
+
+def is_locale_utf8():
+    """Check if the system locale is UTF-8."""
+    lang = os.getenv("LC_CTYPE", "") or os.getenv("LANG", "")
+    return "UTF-8" in lang.upper() or "utf8" in lang.lower()
+
+def supports_emoji():
+    """Detect if the terminal supports emoji output."""
+    if not is_locale_utf8():
+        return False  # Block emoji if UTF-8 is not in locale
+
+    if sys.platform == "win32": # works 32, 64 bits machines
```

Comment:
We can remove this block, we don't run on Windows native only via WSL2

Reply:
humm let's be safe no?

Reply:
But if we want to start windows native, I'd suggest we test enable core functionality first, rather than bells and whistles. I'm trying to be mindful of maintenance

Reply:
We'd need native Windows CI for one, I don't want to add a very minor part of Windows native enablement and give people false expectations of compatibility.

Reply:
We'd need native Windows CI for one, I don't want to add a minor part of Windows native enablement and give people false expectations of compatibility.

Reply:
We'd need native Windows CI for one, I don't want to add a minor part of Windows native enablement and give people false expectations of compatibility.